src/buildHTML.js

```diff
@@ -140,6 +141,8 @@ const shouldHandleSupSub = function(group, options) {
             group.value.alwaysHandleSupSub);
     } else if (group.type === "accent") {
         return isCharacterBox(group.value.base);
+    } else if (group.type === "horizBrace") {
+        return (isSup === group.value.isOver);
```

Comment:
I also like the idea of keeping the logic together. Before I started, the arguments passed to this function were: `shouldHandleSupSub(group.value.base, options)`. So inside the function, `group.value.sub` is unavailable. 

I could pass `group` to the function instead of `group.value.base`. Then all of this bit of logic can be done in one location. Of course, other parts of the function would also need to be tweaked.